arch-snapshot/arch.md

```diff
@@ -51,6 +60,20 @@ details and establish a glossary to ensure consistency within the document.
 
 decomposition into layers. responsibility of the layers.
 
+Layers:
```

Comment:
I think these are not really layers. This is how components are organized into processes.